src/core/ledger/ledger.js

```diff
@@ -44,6 +57,8 @@ type MutableAccount = {|
   balance: G.Grain,
   // The amount of Grain this account has received in past Distributions
   paid: G.Grain,
+  // A history of Grain allocations to the account in chronological order
```

Comment:
Maybe worth mentioning here the merge logic, where an account will have allocations that it received directly or indirectly via merged accounts

Reply:
accepted